Cargo.toml

```diff
@@ -32,3 +32,5 @@ atty = "0.2"
 tracing = "0.1"
 tracing-subscriber = { version = "0.3", features = ["env-filter"] }
 tracing-appender = "0.2"
+clap = {version="4.3.22", features=["derive"]}
```

Comment:
nit: the way the file is formatted, there is spaces before and after `=`, and between `{`, `}` and the content.

Reply:
Just installed a toml formatter extension and let it do its thing on the file e37c742d384a3747f62fc1dc6d57b7d67528ea48 (it also imposed a line length, which is nice I guess)